bin/download_proteins_entrez.py

```diff
@@ -147,7 +147,7 @@ def main(args=None):
     # Process TaxIDs
 
     print("Processing the following taxonomy IDs:")
-    print(taxIds)
+    print(sorted(taxIds))
```

Comment:
Maybe this could be already done in https://github.com/nf-core/metapep/blob/c126e2776e52e0ca63110cd5fc98c8fe3a89571b/bin/download_proteins_entrez.py#L145
for better readability